webpack.config.js

```diff
@@ -1,26 +1,24 @@
 // @ts-check
```

Comment:
Want to drop the style changes on this one; this file tends to get edited a decent amount, so will likely cause annoying merge conflicts.